package io.wildlabs.filemonster.core.exception;

public class ResolveException extends RuntimeException {
    public ResolveException(Throwable cause) {
        super(cause);
    }
}
